tag.sh

```diff
@@ -18,10 +18,11 @@ for dir in ${VERSIONS}; do
   commit_date=$(git show -s HEAD --format=%cd --date=short | sed 's/-//g')
   date_and_hash="${commit_date}-$(git rev-parse --short HEAD)"
 
-  echo "-> Tagging image '$IMAGE_ID' as '$name:$version' and '$name:latest' and '$name:$date_and_hash'"
-  docker tag "$IMAGE_ID" "$name:$version"
-  docker tag "$IMAGE_ID" "$name:latest"
-  docker tag "$IMAGE_ID" "$name:$date_and_hash"
+
+  echo "-> Tagging image '$IMAGE_ID' as '$REGISTRY$name:$version' and '$REGISTRY$name:latest' and '$REGISTRY$name:$date_and_hash'"
```

Comment:
How about creating a new variable (eg. `full_name`) and use it instead of `$REGISTRY$name`. Might  be a bit more readable